source/features/highlight-collaborators-and-own-conversations.tsx

```diff
@@ -53,6 +50,10 @@ void features.add({
 	include: [
 		pageDetect.isRepoConversationList
 	],
+	exclude: [
+		() => select.exists('.blankslate')
```

Comment:
We have this on github-url-detection

Reply:
Only isEmptyRepoRoot

Reply:
We should probably drop that one and just use this function everywhere

Reply:
https://github.com/fregante/github-url-detection/commit/0c36c4c08636174cc039b077680ff90acb3b5d37

Reply:
![](https://media0.giphy.com/media/YFkpsHWCsNUUo/giphy.gif)

Reply:
@fregante how to proceed?

Reply:
`npm deprecate *`